src/calculator/basic.js

```diff
@@ -0,0 +1,31 @@
+import {
+  INPUT_BASIC_DELIMITER,
+  INPUT_STRING_POSITIVE,
+  INPUT_STRING_VALIDATION,
+} from "../constants/error.js";
+import { ERROR } from "../constants/message.js";
+import {
+  isAllNumber,
+  isAllNumberPositive,
+  sumAllNumber,
+} from "../utils/numberUtils.js";
+
+export const basic = (str) => {
+  const numArr = str.split(/[,:]/).map(Number);
+
+  if (numArr[0] === NaN) {
+    throw new Error(ERROR + INPUT_BASIC_DELIMITER);
+  }
+
+  if (!isAllNumber(numArr)) {
+    throw new Error(ERROR + INPUT_STRING_VALIDATION);
+  }
+
+  if (!isAllNumberPositive(numArr)) {
+    throw new Error(ERROR + INPUT_STRING_POSITIVE);
+  }
+
+  if (isAllNumber(numArr) && isAllNumberPositive(numArr)) {
+    return sumAllNumber(numArr);
+  }
```

Comment:
이미 앞에서 `isAllNumber`이 `false`인 경우와 `isAllNumberPositive`가 `false`인 경우를 판별하여 예외 처리를 했기 때문에 여기서는 조건문이 불필요해보입니다!
```suggestion
  return sumAllNumber(numArr);
```

Reply:
그렇네요! 수정했습니다. 짚어주셔서 감사해요!